development/nautobot_config.py

```diff
@@ -43,20 +59,28 @@
     DATABASES["default"]["OPTIONS"] = {"charset": "utf8mb4"}
 
 #
-# Debug
+# Redis
 #
 
-DEBUG = is_truthy(os.getenv("NAUTOBOT_DEBUG", False))
-
-TESTING = len(sys.argv) > 1 and sys.argv[1] == "test"
+# The django-redis cache is used to establish concurrent locks using Redis.
+CACHES = {
+    "default": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": parse_redis_connection(redis_database=0),
+        "TIMEOUT": 300,
+        "OPTIONS": {
+            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+        },
+    }
+}
 
-# Django Debug Toolbar
-DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda _request: DEBUG and not TESTING}
+# Redis Cacheops
+CACHEOPS_REDIS = parse_redis_connection(redis_database=1)
```

Comment:
Shouldn't be needed in a Nautobot-2.0-required version of the plugin.

Reply:
Good catch, we'll fix in the cookie and will get picked up in the next round be drift manager.

Reply:
Removed